packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationFileUploadAdapter.test.ts

```diff
@@ -110,21 +106,12 @@ const createChatContext = (): ChatContext =>
   );
 
 /* @conditional-compile-remove(file-sharing) */
-const generateObservableFileUploads = (quantity: number): ObservableFileUpload[] => {
-  const fileUploads: ObservableFileUpload[] = [];
+const generateFiles = (quantity: number): File[] => {
```

Comment:
super nit (feel free to ignore, this is a test file, but thought it was quite neat), you can remove the for loop and reduce this to one line using an array generator!
```ts
const generateFiles = (quantity: number): File[] =>
  Array.from({ length: quantity }, () => new File([], nanoid(), { type: 'text/plain' }));
```

Reply:
Neat. Thanks